### ISSUE TITLE
Loading logo WGLMakie page

### ISSUE THREAD

**Issue by AlexisRenchon** [JuliaPlots/Makie.jl#2191]
It would be great to have a loading logo while a WGLMakie page loads. 
For example, https://cupoftea.earth/carbvis/, just shows a blank page for 10 seconds until the figure finally appears. 
It would be helpful to have a loading logo to warn page visitors that there is a "work in progress". 
Currently, each time I share a link with colleagues, I warn them "the page takes about 10 seconds to load", so that they don't close it early thinking it is not working... 
Might be quite easy to implement, big reward?

**Comment by lazarusA** [comment]
I  think is more a css thing than a Makie thing. I did it  [here](https://lazarusa.github.io/ClimateVisualisations/app_tau.html) adding some css styling and some minor. (Sorry, at the moment I don't have a minimal thing that you can copy/paste).

**Comment by AlexisRenchon** [comment]
I see. WGLMakie / JSServe is generating html with css and js though, so these could be added? 
I understand it should be optional, you probably don't want to add it for a quick loading figure. 

(BTW, really cool stuff @lazarusA https://github.com/lazarusA/ClimateVisualisations. Makes me think of https://cupoftea.earth/ by me and https://this-is-fine.org/ by @SimonDanisch. Would be cool to join our effort some day.)

**Comment by lazarusA** [comment]
yeah, is mainly based on Simon's work. Specially the css tricks. And for now, that repo is just an external server for the main webpage here: https://www.bgc-jena.mpg.de/en/bgi/gallery I will be adding more makie stuff here and there on a weekly/monthly basis.

**Comment by lazarusA** [comment]
Ok. Now there is a skeleton to do an app with a loading logo  [here](https://github.com/lazarusA/ClimateVisualisations/tree/main/src/healpix_app). Note: **slider1 [not working].**

Output [here](https://lazarusa.github.io/ClimateVisualisations/appt2m_healpix.html)

Feel free to recommend (cite) the site 😄. 

Maybe close the issue if this solves your problem.
